aiidalab_launch/util.py

```diff
@@ -43,6 +44,13 @@ def spinner(
     def spin() -> None:
         if msg:
             click.echo(f"{msg.rstrip()} ", nl=False, err=True)
+
+        # Don't show spinner if verbose output is enabled
+        level = logging.getLogger().getEffectiveLevel()
+        if level != 0 and level < 40:
```

Comment:
I am not very experienced with this particular code, but the `level < 40` condition seems strange. If a higher number means "more verbose", then there should be no upper limit, no?

Reply:
@danielhollas Is the user allowed to change the level? If I understand correctly, the level is increased by adding more `v`? If this is the case, it is better to update the help information of `-v`.

Reply:
@yakutovicha actually, the level indicates severity, so lower number means less severe, hence more verbose. See

https://docs.python.org/3/library/logging.html#logging-levels

The default level is `logging.ERROR` which is 40. I have replaced the hardcoded numbers with the proper variables from the logging module, thanks for pointing this out.

Reply:
> Is the user allowed to change the level? If I understand correctly, the level is increased by adding more v? If this is the case, it is better to update the help information of -v.

@unkcpz yes, you can adjust the level based on the number of `v`s. The current help says

> Provide this option to increase the output verbosity of the launcher.

@yakutovicha @unkcpz do you have suggestion how to improve this? I cannot figure out how to word this concisely.

Reply:
I am sure no good at wording docs. I check the python cmdline doc https://docs.python.org/3/using/cmdline.html#cmdoption-1. We can assume the user only uses `-v` and `-vv` and for `-vv` giving the detail of what will be printed more.